node/standalone/src/chain_spec.rs

```diff
@@ -147,8 +147,8 @@ fn testnet_genesis(
 			changes_trie_config: Default::default(),
 		}),
 		pallet_balances: Some(BalancesConfig {
-			// Configure endowed accounts with initial balance of 1 << 60.
-			balances: endowed_accounts.iter().cloned().map(|k|(k, 1 << 60)).collect(),
+			// Configure endowed accounts with initial balance of 1000000 << 64.
```

Comment:
I feel it doesn't make sense to increase the shift and the number itself. Both are doing the same thing,
Either do:
`1 << 70` or `10000000000000000 << 60` (I prefer the first)

Reply:
I've modified the shift so its binary 1 shifted 80 times. The genesis account will now have 1208925.819614629174706176 tokens when initiated.